src/cpu/README.md

```diff
@@ -44,6 +44,7 @@ enable or disable parts of code. There the following macros defined:
 - `DNNL_X64` is 1 on x64 architecture;
 - `DNNL_AARCH64` is 1 on Arm AArch64 architecture;
 - `DNNL_PPC64` is 1 on OpenPOWER / IBM Power architecture;
+- `DNNL_S390X` is 1 on IBMz / s390x architecture;
```

Comment:
One thing I wanted to confirm is whether `S390X` is the right name for this configuration. Based on my limited knowledge of IBM architecture something like `IBMZ` should be more appropriate.

Reply:
IBMz is the current name of the family of systems that use the Z Architecture, not the name of the architecture itself.  The Architecture now is named z/Architecture. Linux, GNU configure, etc. all use `s390x-ibm-*` as the target triplet.  I will reconfirm the choice, but with the use of s390x throughout the Open Source Software ecosystem, s390x seems more consistent.

Reply:
Thank you for the clarifications!

Reply:
I double-checked and S390X name is fine.